producer-rest-core/src/main/scala/za/co/absa/spline/producer/rest/controller/StatusController.scala

```diff
@@ -42,10 +44,24 @@ class StatusController @Autowired()(
     new ApiResponse(code = 200, message = "Everything's working"),
     new ApiResponse(code = 503, message = "There is a problem")
   ))
-  def status(response: HttpServletResponse): Future[_] = repo
+  def statusHead(response: HttpServletResponse): Future[_] = repo
     .isDatabaseOk
     .map({
       if (_) new ResponseEntity(HttpStatus.OK)
       else new ResponseEntity(HttpStatus.SERVICE_UNAVAILABLE)
     })
+
+  @RequestMapping(
+    path = Array("/status"),
+    method = Array(RequestMethod.GET))
+  @ApiOperation(
+    value = "Server status",
+    notes = "Returns server status, version and abilities")
+  def statusGet(response: `HttpServletResponse`): Future[ServerStatus] = repo
+    .isDatabaseOk
+    .map({
+      if (_) "OK"
```

Comment:
`GET` and `HEAD` should have the same contract in terms of response codes and headers.
See: https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.4